framework/src/application/schema/application_config_schema.js

```diff
@@ -299,13 +299,35 @@ module.exports = {
 						transactions: {
 							type: 'object',
 							properties: {
-								maxTransactionsPerQueue: {
+								maxTransactions: {
 									type: 'integer',
-									minimum: 100,
-									maximum: 5000,
+									maximum: 4096,
+								},
+								maxTransactionsPerAccount: {
+									type: 'integer',
+									maximum: 64,
+								},
+								transactionExpiryTime: {
+									type: 'integer',
+									minimum: 10800000,
+									maximum: 10800000,
+								},
+								minEntranceFeePriority: {
+									type: 'integer',
+									minimum: 1,
+								},
+								minReplacementFeeDifference: {
+									type: 'integer',
+									minimum: 10,
```

Comment:
`minReplacementFee` starts with 10 rite?

Reply:
default to 10, but it can start even from `0`

Reply:
🤔 if it starts from 0 then `minEntranceFee` is 1 correct? this doesnt make sense to me.. `minReplacementFee` must be atleast greater than `minEntranceFee`?

Reply:
minReplacementFeeDifference is if u have fee 10 beddow transaction, in order to replace, you need to set the fee to at least 20 beddow. So, it is not related to the entranceFee.

Actually setting 0 will not make sense because in order to replace the transaction, it has to have higher fee than original. Therefore, in this case `1` will be the correct minimum value (but it is set to 10 by default in library